### PR TITLE
[5.2] Usability: Back-end - Add total number of items to Page Navigation 

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue ##43548 .

### Summary of Changes
Adds the total number of items at the end of a table

image

The total will be present on any list view that uses `<?php echo $this->pagination->getListFooter(); ?>`

### Testing Instructions
Apply PR and check the bottom of the table
Make sure that the total is correct after filters have been applied
Make sure that the total is displayed even when the pagination is not applied

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: layouts/joomla/pagination/links.php [hunk 1]
```diff
@@ -45,40 +46,39 @@
...
+        <?php echo Text::sprintf('JGLOBAL_ITEM_COUNT', $total); ?>
+    </div>
+    <div class="pagination pagination-toolbar text-center mt-0">
+        <?php if ($showLimitBox) : ?>
+            <div class="limit float-end">
+                <?php echo Text::_('JGLOBAL_DISPLAY_NUM') . $list['limitfield']; ?>
+            </div>
+        <?php endif; ?>
+
+        <?php if ($showPagesLinks && (!empty($pages))): ?>
```
Comment (Quy) [inline layouts/joomla/pagination/links.php]:
```suggestion
        <?php if ($showPagesLinks && !empty($pages)) : ?>
```

**Comment by HLeithner** [comment]
if you already introduce a new line, wouldn't it make sense to also show the current item position something like:
`1 - 20 / 60 items`

Language string would be something like this:

JLIB_HTML_PAGINATION_ITEMS="%1$d - %2$d / %3$d items"

**Comment by brianteeman** [comment]
Note that both this implementation and the alternative are not completely correct when it comes to accessibility. The information should really be in the caption element

File: layouts/joomla/pagination/links.php [hunk 2]
```diff
@@ -45,40 +46,39 @@
...
+        <?php echo Text::sprintf('JGLOBAL_ITEM_COUNT', $total); ?>
+    </div>
+    <div class="pagination pagination-toolbar text-center mt-0">
+        <?php if ($showLimitBox) : ?>
+            <div class="limit float-end">
+                <?php echo Text::_('JGLOBAL_DISPLAY_NUM') . $list['limitfield']; ?>
+            </div>
+        <?php endif; ?>
+
+        <?php if ($showPagesLinks && !empty($pages)): ?>
```
Comment (Quy) [inline layouts/joomla/pagination/links.php]:
```suggestion
        <?php if ($showPagesLinks && !empty($pages)) : ?>
```

File: administrator/language/en-GB/joomla.ini [hunk 3]
```diff
@@ -483,6 +483,7 @@ JGLOBAL_INHERIT="Inherit"
 JGLOBAL_INTEGRATION_LABEL="Integration"
 JGLOBAL_INTRO_TEXT="Intro Text"
 JGLOBAL_ISFREESOFTWARE="%s is free software released under the <a href=\"https://www.gnu.org/licenses/gpl-2.0.html\" target=\"_blank\" rel=\"noopener noreferrer\">GNU General Public License</a>."
+JGLOBAL_ITEM_COUNT="%d Items"
```
Comment (Quy) [inline administrator/language/en-GB/joomla.ini:486]: Needs to handle when there is only one item.

**Comment by brianteeman** [comment]
Closing this in favour of #43575

**Comment by pe7er** [comment]
Thanks @brianteeman !

**Pull request closed without merging**